### PR TITLE
Update Kafka readme regarding persistency

### PR DESCRIPTION
**Pull request by davidhadas**
Add that ZooKeeper persistency is separately configured

**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

**Additional information**

### CODE REVIEW

**Comment by carrodher** [comment]
Thanks for the contribution, before continue, you should increment the chart version in the `Chart.yml` according to [semver](http://semver.org/):

> Given a version number `MAJOR.MINOR.PATCH`, increment the:
> - MAJOR version when you make incompatible API changes,
> - MINOR version when you add functionality in a backwards-compatible manner, and
> - PATCH version when you make backwards-compatible bug fixes.
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

**Comment by davidhadas** [comment]
done

File: bitnami/kafka/README.md [hunk 1]
```diff
@@ -155,6 +155,7 @@ The following tables lists the configurable parameters of the Kafka chart and th
 | `metrics.jmx.jmxPort`                   | The jmx port which JMX style metrics are exposed (note: these are not scrapeable by Prometheus)           | `5555`                                                             |
 | `metrics.jmx.whitelistObjectNames`      | Allows setting which JMX objects you want to expose to via JMX stats to JMX Exporter                      | (see `values.yaml`)                                                |
 | `zookeeper.enabled`                     | Switch to enable or disable the Zookeeper helm chart                                                      | `true`                                                             |
+| `zookeeper.persistence.enabled`         | Enable Zookeaper persistence using PVC                                                                    | `true`                                                             |
```
Comment (juan131) [inline bitnami/kafka/README.md]: Typo:
```diff
- ... Zookeaper persistence using PVC ...
+ ... Zookeeper persistence using PVC ...
```

File: bitnami/kafka/README.md [hunk 2]
```diff
@@ -109,7 +109,7 @@ The following tables lists the configurable parameters of the Kafka chart and th
 | `service.nodePort`                      | Kubernetes Service nodePort                                                                               | `nil`                                                              |
 | `service.loadBalancerIP`                | loadBalancerIP for Kafka Service                                                                          | `nil`                                                              |
 | `service.annotations`                   | Service annotations                                                                                       | ``                                                                 |
-| `persistence.enabled`                   | Enable persistence using PVC                                                                              | `true`                                                             |
+| `persistence.enabled`                   | Enable Kafka persistence using PVC, note that zookeaper perisstency is unaffected                        | `true`                                                             |
```
Comment (juan131) [inline bitnami/kafka/README.md]: Typo:
```diff
- ... zookeaper perisstency ...
+ ... Zookeeper persistence ...
```

Comment (davidhadas) [reply bitnami/kafka/README.md]: done

**Pull request merged**